Formula/victoriametrics.rb

```diff
@@ -0,0 +1,63 @@
+class Victoriametrics < Formula
+  desc "Cost-effective and scalable monitoring solution and time series database"
+  homepage "https://victoriametrics.com/"
+  url "https://github.com/VictoriaMetrics/VictoriaMetrics/archive/v1.88.1.tar.gz"
+  sha256 "92803ae61f927b7173e25d4537c429bf3054edfa2f719bafc8fb9fcd411cab87"
+  license "Apache-2.0"
+
+  depends_on "go" => :build
+  depends_on "make" => :build
+
+  def install
+    system "make", "victoria-metrics"
+    bin.install "bin/victoria-metrics"
+
+    (etc/"victoriametrics/scrape.yml").write <<~EOS
+      global:
+        scrape_interval: 10s
+
+      scrape_configs:
+        - job_name: "victoriametrics"
+          static_configs:
+          - targets: ["127.0.0.1:8428"]
+    EOS
```

Comment:
Should upstream provide something for this?

Reply:
VIctoriaMetrics will start but without this file it will not scrape any data like self monitoring. This file is like prometheus.yml - see https://github.com/Homebrew/homebrew-core/blob/master/Formula/prometheus.rb#L51-L59

Reply:
fair enough.